### PR TITLE
LPD-22019 Integration test for LPD-21799

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-22019](https://liferay.atlassian.net/browse/LPD-22019)

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java [hunk 1]
```diff
@@ -873,6 +892,82 @@ public void testGetNotifiableUsers() throws Exception {
...
+			userAccountAdmin.getUserId(), accountEntry.getAccountEntryGroupId(),
+			new long[] {roleAccountAdmin.getRoleId()});
+
+		CommerceCurrency commerceCurrency =
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getCompanyId());
+
+		CommerceChannel commerceChannel =
+			CommerceChannelLocalServiceUtil.addCommerceChannel(
+				null, AccountConstants.ACCOUNT_ENTRY_ID_DEFAULT,
+				_group.getGroupId(), "Test Channel",
```
Comment (thallescezar) [inline modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java:938]: Can you change this name to a random String?

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Review by thallescezar — APPROVED** [review]

**Comment by thallescezar** [comment]
Approved!

**Comment by linolaoj** [comment]
ci:forward

**Review by linolaoj — APPROVED** [review]

**Pull request closed without merging**